src/user/editor_payout_tasks.py

```diff
@@ -0,0 +1,182 @@
+from calendar import monthrange
+from io import StringIO
+import csv
+import datetime
+import json
+import math
+import requests
+
+from celery.decorators import periodic_task
+from celery.task.schedules import crontab
+from django.contrib.contenttypes.models import ContentType
+from django.core.mail import EmailMessage
+
+from hub.models import Hub
+from reputation.distributor import Distributor
+from reputation.distributions import Distribution  # this is NOT the model
+from researchhub_access_group.constants import EDITOR
+from purchase.related_models.constants.rsc_exchange_currency import USD
+from purchase.related_models.rsc_exchange_rate_model import RscExchangeRate
+from researchhub.settings import APP_ENV, MORALIS_API_KEY, WEB3_RSC_ADDRESS
+from user.constants.gatekeeper_constants import (
+    EDITOR_PAYOUT_ADMIN, PAYOUT_EXCLUSION_LIST
+)
+from user.related_models.gatekeeper_model import Gatekeeper
+from user.related_models.user_model import User
+from utils import sentry
+
+UNI_SWAP_BUNDLE_ID = 1  # their own hard-coded eth-bundle id
+UNI_SWAP_GRAPH_URI = "https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2"
+USD_PAY_AMOUNT_PER_MONTH = 3000
+USD_PER_RSC_PRICE_FLOOR = .033
+
+MORALIS_LOOKUP_URI = "https://deep-index.moralis.io/api/v2/erc20/{address}/price".format(address=WEB3_RSC_ADDRESS)
+
+
+@periodic_task(
+    run_every=crontab(hour=15, minute=0),  # 3PM System Time (PST)
+    priority=1,
+    options={'queue': f'{APP_ENV}_core_queue'}
+)
+def editor_daily_payout_task():
+    today = datetime.date.today()
+    num_days_this_month = monthrange(today.year, today.month)[1]
+    result = get_daily_rsc_payout_amount_from_deep_index(num_days_this_month)
+
+    # Keeping record of exchange rate used today
+    RscExchangeRate.objects.create(
+      rate=result['rate'],
+      target_currency=USD,
+    )
+
+    excluded_user_email = Gatekeeper.objects.filter(
+        type__in=[EDITOR_PAYOUT_ADMIN, PAYOUT_EXCLUSION_LIST]
+    ).values_list('email', flat=True)
+
+    editors = User.objects.filter(
+        permissions__isnull=False,
+        permissions__access_type=EDITOR,
+        permissions__content_type=ContentType.objects.get_for_model(Hub)
+    ).distinct().exclude(email__in=(excluded_user_email))
+
+    csv_prep = {
+      'amount-rsc': [],
+      'emails': [],
+      'names': [],
+      'usd-rsc-rate': [],
+    }
+
+    for editor in editors.iterator():
+        try:
+            pay_amount = result['pay_amount']
+            distributor = Distributor(
+                  # this is NOT the model. It's a simple object
+                  Distribution('EDITOR_PAYOUT', pay_amount, False),
+                  editor,
+                  None,
+                  today
+              )
+            distributor.distribute()
```

Comment:
if you don't run distribute here -- or make it a feature flag then the payout will not happen.